tests/components/openuv/test_config_flow.py

```diff
@@ -39,66 +24,38 @@ async def test_duplicate_error(hass):
         CONF_LONGITUDE: -104.9812612,
     }
 
-    MockConfigEntry(domain=DOMAIN, data=conf).add_to_hass(hass)
-    flow = config_flow.OpenUvFlowHandler()
-    flow.hass = hass
-
-    result = await flow.async_step_user(user_input=conf)
-    assert result["errors"] == {CONF_LATITUDE: "identifier_exists"}
-
-
-@pytest.mark.parametrize("uv_index_response", [mock_coro(exception=OpenUvError)])
-async def test_invalid_api_key(hass, mock_pyopenuv):
-    """Test that an invalid API key throws an error."""
-    conf = {
-        CONF_API_KEY: "12345abcde",
-        CONF_ELEVATION: 59.1234,
-        CONF_LATITUDE: 39.128712,
-        CONF_LONGITUDE: -104.9812612,
-    }
-
-    flow = config_flow.OpenUvFlowHandler()
-    flow.hass = hass
+    MockConfigEntry(
+        domain=DOMAIN, unique_id="39.128712, -104.9812612", data=conf
+    ).add_to_hass(hass)
 
-    result = await flow.async_step_user(user_input=conf)
-    assert result["errors"] == {CONF_API_KEY: "invalid_api_key"}
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": SOURCE_USER}, data=conf
+    )
 
+    assert result["type"] == data_entry_flow.RESULT_TYPE_ABORT
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5596821)